io.github.hakuneko.HakuNeko.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.hakuneko.HakuNeko</id>
+	<launchable>io.github.hakuneko.HakuNeko.desktop</launchable>>
+	<name>HakuNeko<name>
+	<project_license>Unlicense</project_license>
+	<developer_name>Ronny Wegener</developer_name>
+  	<summary>Manga & Anime Downloader for Linux, Windows & MacOS</summary>
```

Comment:
Not sure if mentioning other platforms is worth it ?

Reply:
I changed the summary to be more descriptive and removed mentions of other platforms.